contracts/src/SafeInternationalHarbour.sol

```diff
@@ -30,7 +34,7 @@ import "./mixins/QuotaMixin.sol";
  * @dev The {SignatureStored} event is the only hook required by indexers; however, the contract is
  *      fully functional without any off‑chain infrastructure.
  */
```

Comment:
One thing that triggered my OCD was that we mix NapSpec styles in the contracts - sometimes it's `/** */` and sometimes `///`. 

I personally think the latter looks slightly better, but only because I coded Rust for so long 😛. Im fine with either but would prefer to stick to one style.

Same applies for formatting styles (some places align parameter descriptions, some don't , some places have extra new lines between the NapSpec sections - which I like for readability - others don't, etc.)

Reply:
how annoying ... will try to fix this everywhere